packages/zosuss/__tests__/__system__/Shell.system.test.ts

```diff
@@ -149,7 +149,7 @@ describe("zowe uss issue ssh api call test", () => {
             expect(error.toString()).toContain(ZosUssMessages.connectionRefused.message);
         } else {
             expect(error.toString().includes(ZosUssMessages.allAuthMethodsFailed.message) ||
-                   error.toString().includes(ZosUssMessages.connectionRefused.message));
+                   error.toString().includes(ZosUssMessages.connectionRefused.message)).toBe(true);
```

Comment:
general question:
isn't there some sort of rule that prevents logic from being added as part of the expected value?

Reply:
Have looked, but not sure that I see one here: https://github.com/jest-community/eslint-plugin-jest/tree/main/docs/rules

Reply:
Although we tend not to make tests too complext, adding [`no-if`](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-if.md) will probably be too much (and it still may not address this particular problem) 😋